src/AccessibilityInsights.Rules/PropertyConditions/ElementGroups.cs

```diff
@@ -144,6 +145,15 @@ private static Condition CreateParentWPFDataItemCondition()
             return Condition.Create(IsParentWPFDataItem);
         }
 
+        private static Condition CreateWPFScrollBarPageUpOrPageDownButtons()
```

Comment:
This name will get pretty wordy once PageLeft and PageRight are added. Maybe CreateWPFScrollBarPageScrollButtons?